src/configure/configure.ts

```diff
@@ -450,14 +450,34 @@ class PipelineConfigurer {
 
         // show available resources and get the chosen one
         this.appServiceClient = new AppServiceClient(this.inputs.azureSession.credentials, this.inputs.azureSession.tenantId, this.inputs.azureSession.environment.portalUrl, this.inputs.targetResource.subscriptionId);
+        
+        let resourceArray: Promise<Array<{label: string, data: GenericResource}>> = null;
+        let selectAppText: string = "";
+        let placeHolderText: string = "";
+
+        switch(this.inputs.pipelineParameters.pipelineTemplate.targetType) {
+            case TargetResourceType.WebApp:
+                resourceArray = this.appServiceClient.GetAppServices(this.inputs.pipelineParameters.pipelineTemplate.targetKind)
+                    .then((webApps) => webApps.map(x => { return { label: x.name, data: x }; }));
+                selectAppText = constants.SelectFunctionApp;
+                placeHolderText = Messages.selectFunctionApp;
```

Comment:
This would not work when the app type to be used is web app. We can't have specific string here.

You can either use a generic message for place holder like: Select app 
or based on targetType, you would need to assign the right place holder

Reply:
I've solved for this case. We'll show specific resource as text only.